packages/vue/src/components/ReactiveBase/index.jsx

```diff
@@ -201,6 +204,12 @@ const ReactiveBase = {
 			if (this.$props.transformResponse) {
 				appbaseRef.transformResponse = this.$props.transformResponse;
 			}
+			const parsedUrl = this.url && this.url.replace(/\/\/.*@/, '//');
+			this.analyticsRef = AppbaseAnalytics.init({
+				index: appbaseRef.app,
+				credentials: appbaseRef.credentials,
+				url: parsedUrl,
```

Comment:
@mohdashraf010897 Can you handle `endpoint` here for analytics, only use the host part of `endpoint` if present?